pkg/config/config.go

```diff
@@ -42,6 +42,8 @@ type Config struct {
 type WebConfig struct {
 	Maintenance   bool       `env:"TRISA_MAINTENANCE" desc:"if true sets the web UI to maintenance mode; inherited from parent"`
 	Enabled       bool       `default:"true" desc:"if false, the web UI server will not be run"`
+	APIEnabled    bool       `default:"true" split_words:"true" desc:"if false, the API server will return unavailable when accessed; subordinate to the enabled flag"`
+	UIEnabled     bool       `default:"true" split_words:"true" desc:"if false, the UI server will return unavailable when accessed; subordinate to the enabled flag"`
```

Comment:
If `Enabled` is false, then both API and UI are disabled. If `Enabled` is `true` then and only then will `APIEnabled` or `UIEnabled` be used. Note that if both `APIEnabled` and `UIEnabled` are `false`, then an invalid config error is returned.